### DISCUSSION TITLE
Polish translation

### DISCUSSION THREAD

**Discussion by 000AXR** [OrcaSlicer/OrcaSlicer#2100] [Dev]
hello I would like to help translate a text into Polish language
I have already started translating in Poedit.
but I don't know how to check in real time in the interface  
so that these translations make sense and have context .
(I am not a programmer)

edit. 

 I translated everything and now I would like to add a new language and test locally and make changes and corrections 
in the translated text .
 How to do this I don't understand this instruction or it is not accurate . 
someone could help me with this .
also is it possible when testing the language appeared id of each string so that I can more easily make corrections .
I use Win11

**Comment by ptelman** [comment]
@000AXR what is the result of your work?
Maybe you could create a PR with the file(s) you created?
